spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/actions/TestRemoveOrphanFilesAction.java

```diff
@@ -138,8 +138,7 @@ public void testDryRun() throws IOException, InterruptedException {
     invalidFiles.removeAll(validFiles);
     Assert.assertEquals("Should be 1 invalid file", 1, invalidFiles.size());
 
-    // sleep for 1 second to unsure files will be old enough
-    Thread.sleep(1000);
+    waitUntilAfter(System.currentTimeMillis());
```

Comment:
I think this will jsut immediately return. The `waitUntilAfter` function loops while the passed in timestamp is less than `System.currentTimeMillis`.

Other places I see `waitUntilAfter` used tend to wait until `table.currentSnapshot.timestampMillis()`. But in this file it does seem that this is how `waitUntilAfter` is used, so I'm ok with this.

If we notice this being flakey, then we might want to change it to using the table's current snapshot timestamp instead.